webpack/redux/actions/RedHatRepositories/helpers.js

```diff
@@ -14,25 +14,23 @@ const recommendedRepositoriesRHEL = [
   'rhel-8-for-x86_64-baseos-kickstart',
   'rhel-8-for-x86_64-appstream-rpms',
   'rhel-8-for-x86_64-appstream-kickstart',
+  'rhel-8-for-x86_64-baseos-eus-rpms',
+  'rhel-8-for-x86_64-appstream-eus-rpms',
   'rhel-7-server-rpms',
   'rhel-7-server-optional-rpms',
   'rhel-7-server-extras-rpms',
-  'rhel-7-server-eus-rpms',
   'rhel-7-server-kickstart',
-  'rhel-6-server-els-rpms',
-  'rhel-6-server-kickstart',
 ];
 
 const recommendedRepositoriesSatTools = [
-  'satellite-tools-6.10-for-rhel-8-x86_64-rpms',
-  'rhel-7-server-satellite-tools-6.10-rpms',
-  'rhel-6-server-els-satellite-tools-6.10-rpms',
+  'satellite-tools-6.11-for-rhel-8-x86_64-rpms',
+  'rhel-7-server-satellite-tools-6.11-rpms',
```

Comment:
need to confirm that tools repos are 6.11 vs 6, will check

Reply:
These should be:
```
satellite-client-6-for-rhel-8-x86_64-rpms
rhel-7-server-satellite-client-6-rpms
```

Reply:
and i assume:  rhel-6-server-els-satellite-client-6-rpms

Reply:
@jlsherrill crosschecked the doc you sent me and updated, this should be correct now.